pkg/bff/api/v1/api.go

```diff
@@ -23,9 +22,16 @@ type Reply struct {
 	Error   string `json:"error,omitempty" yaml:"error,omitempty"`
 }
 
+// StatusParams is parsed from the query parameters of the GET request
+type StatusParams struct {
+	NoGDS bool `url:"nogds,omitempty" form:"nogds" default:"false"`
+}
+
 // StatusReply is returned on status requests. Note that no request is needed.
 type StatusReply struct {
-	Status    string    `json:"status"`
-	Timestamp time.Time `json:"timestamp,omitempty"`
-	Version   string    `json:"version,omitempty"`
+	Status  string `json:"status"`
+	Uptime  string `json:"uptime,omitempty"`
+	Version string `json:"version,omitempty"`
+	TestNet string `json:"testnet,omitempty"`
+	MainNet string `json:"mainnet,omitempty"`
```

Comment:
TestNet and MainNet will be one of:

- "unavailable"
- "healthy" 

☝🏽  the two most common options

- "danger"
- "unhealthy"
- "maintenance"

Reply:
"unavailable" is added when we cannot connect to GDS; the others are strings to lower case of the ServiceStatus enum in the TRISA protocol buffers. 

Unfortunately, version information for GDS is not available since it is not part of the TRISA protocol buffers.